ext/pdo_dblib/tests/types.phpt

```diff
@@ -64,3 +69,5 @@ bool(true)
 bool(true)
 bool(true)
 bool(true)
+bool(true)
+bool(true)
```

Comment:
Ah, my test VM had the older version set. Looks fine now. I think ideally we would skip this test for older TDS versions, but I don't think we can detect that from PHP/PDO. Maybe just add a comment saying why this test might fail?